src/modules/rlm_rest/rlm_rest.c

```diff
@@ -1167,6 +1167,13 @@ static int mod_thread_instantiate(module_thread_inst_ctx_t const *mctx)
 		return -1;
 	}
 
+	if (fr_rest_slab_list_alloc(t, &t->slab, mctx->el, inst->conn_config.interval, inst->conn_config.min_elements,
```

Comment:
This many arguments really indicates there should be a configuration structure containing the configurable elements of the slab allocator.   Then you could just nest it inside the generic curl config structure, and not have to pass in five arguments manually.